assets/js/base/components/order-select/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Label from '../label';
+import './style.scss';
+
+const OrderSelect = ( { className, componentId, defaultValue, label, onChange, options, screenReaderLabel, readOnly, value } ) => {
```

Comment:
What is OrderSelect?  I think it would be helpful to include some doc block here explaining what the `OrderSelect` component is used for.

Reply:
Done in b7ed466dc46cd15c91d4a1984d6d76dfc803c297.